libs/common-services/src/main/java/uk/gov/di/ipv/core/library/config/CoreFeatureFlag.java

```diff
@@ -1,10 +1,7 @@
 package uk.gov.di.ipv.core.library.config;
 
 public enum CoreFeatureFlag implements FeatureFlag {
-    USE_CONTRA_INDICATOR_VC("useContraIndicatorVC"),
-    USE_POST_MITIGATIONS("usePostMitigations"),
-    MITIGATION_ENABLED("mitigationEnabled"),
-    BUNDLE_CIMIT_VC("bundleCimitVC");
+    UNUSED_PLACEHOLDER("unusedPlaceHolder");
```

Comment:
where is this enum used ?

Reply:
It's only used in tests - it's just a placeholder so that the enum class itself can still exist. We also have a couple of tests that rely on a feature flag enum existing, so this feeding into them.

Reply:
this class can be removed if not needed.